src/js/_store/modules/categories.js

```diff
@@ -3,21 +3,60 @@ import axios from '@Helpers/axiosDefault';
 export default {
   namespaced: true,
   state: {
+    disabled: false,
     loading: false,
```

Comment:
[imo]

「作成中」というローディング状態を明示的にした方がわかりやすいかなと思うので、disabled は削除して loading の state を使用して「作成中」と表示できるように修正してみてください。
全体的に記述もシンプルになるかなと思います！

```suggestion
    loading: false,
```

Reply:
disabledではなくloadingを使用するように変更しました！
d02397244291d6f9ba19f7f5751278b3230e07f0